components/ui/select/index.tsx

```
@@ -0,0 +1,92 @@
+      style={{
+        flex: 1,
+        flexDirection: "row",
+        flexWrap: "wrap",
+        gap: 8,
+      }}
+    >
+      {
+        items.map((item) => (
+          <Pressable
+            onPress={() => handleSelect(item)}
+          >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)